src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/transactionDataListItemStructure.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+    AvatarIcon,
+    DataList,
+    Heading,
+    IconType,
+    NumberFormat,
+    Spinner,
+    formatterUtils,
+    type AvatarIconVariant,
+} from '../../../../../core';
+import { QueryType, createBlockExplorerLink } from '../../../../utils/blockExplorerUtils/blockExplorerUtils';
+import { formatDate } from '../../../../utils/timestampUtils';
+import {
+    TransactionType,
+    TxStatusCode,
+    txHeadingStringList,
+    txIconTypeList,
+    txVariantList,
+    type ITransactionDataListItemProps,
+} from './transactionDataListItemStructure.api';
+
+export const TransactionDataListItemStructure: React.FC<ITransactionDataListItemProps> = (props) => {
+    const {
+        chainId,
+        tokenAddress,
+        tokenSymbol,
+        tokenValue,
+        usdEstimate,
+        txType,
+        txStatus = TxStatusCode.PENDING,
+        unixTimestamp,
+        txHash,
+        ...otherProps
+    } = props;
+
+    const getEffectiveStatus = () => {
+        const type = txType;
+        if (txStatus === TxStatusCode.FAILED) {
+            return {
+                icon: IconType.CLOSE,
+                variant: 'critical' as AvatarIconVariant,
+                heading: 'Failed transaction',
+            };
+        }
+
+        return {
+            icon: type ? txIconTypeList[type] : IconType.HELP,
+            variant: type ? txVariantList[type] : ('neutral' as AvatarIconVariant),
+            heading: type ? txHeadingStringList[type] : 'Unknown',
+        };
+    };
+
+    const { icon, variant, heading } = getEffectiveStatus();
+
+    const getTxIcon = () => {
+        if (txStatus !== TxStatusCode.PENDING) {
+            return <AvatarIcon className="shrink-0" variant={variant} icon={icon} responsiveSize={{ md: 'md' }} />;
+        }
+        return (
+            <div className="flex size-6 shrink-0 items-center justify-center md:size-8">
+                <Spinner className="transition" variant="neutral" responsiveSize={{ md: 'lg' }} />
+            </div>
+        );
+    };
+
+    return (
+        <DataList.Item
+            className="min-w-fit !py-0 px-4 md:px-6"
+            {...otherProps}
+            href={createBlockExplorerLink({ chainId, queryType: QueryType.TX, txHash })}
+            target="_blank"
+        >
+            <div className="flex w-full justify-between py-3 md:py-4">
```

Comment:
This isn't really space between as much as gap-y-3 and gap-y-4 (on md). Of course visually it works, but I'm highlighting in case it's relevant when truncating on small screens. The date, token value and symbol are wrapping on smaller screens